crates/rbuilder/src/building/order_commit.rs

```diff
@@ -181,19 +181,25 @@ where
         &mut self.db
     }
 }
-
+/// Common data used by TransactionOk/BundleOk
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct TransactionExecutionInfo {
+    pub tx: TransactionSignedEcRecoveredWithBlobs,
+    pub receipt: Receipt,
+    pub gas_used: u64,
+    /// On loss capped to 0
```

Comment:
I thing this change may introduce bugs and issues in the future.

Before we only put coinbase profit for order as a whole but now we do it per tx and we don't count negative profits (such as  refund txs profits).

It was OK to put U256 to order coinbase profit because we disallow orders with negative profits overall so it would be impossible to have negative value there on the other hand its completely fine to have txs with negative profits.

I think it should be I256 here because this type is more fit here than 256. IIRC we have some methods in utils to work with u256/i256 since its pretty underdeveloped in alloy.